### PR TITLE
Update How-To-Host-Feed.md

### PR DESCRIPTION
**Pull request by scott64bt**
Updating ProGet requirements

### CODE REVIEW

File: How-To-Host-Feed.md [hunk 1]
```diff
@@ -332,11 +332,11 @@ See https://help.sonatype.com/repomanager3/high-availability-introduction for de
...
 * Windows Server
-* CPU - 4 cores (Recommend more)
-* RAM - 8GB+
-* Storage (HDD) - We recommend 2-5TB of free space for a local file store of artifacts (default install needs 20GB).
-* Database: SQL Server. Assume high specs for SQL Server, pretty much the highest specs you can configure on a Windows machine. 16+ cores (CPU), 64GB+ RAM, 5TB+, etc
-* Chocolatey Repository Type: Chocolatey or NuGet
+* CPU - 2 cores (4+ cores recommended for large instances)
+* RAM - 4GB+ (8GB recommended for large instances)
+* Storage (HDD) - We recommend 2-5TB of free space for a local file store of artifacts (default install needs 10GB).
+* Database: Database: SQL Server Express or better. Install available during download. Minimum specs 1GB RAM, 6GB disk space, higher specs may be required if creating more than a simple feed repository.
```
Comment (gep13) [inline How-To-Host-Feed.md]: @scott64bt as @ferventcoder mentioned in your other PR, can you please remove the mention of SQL Express here?

**Comment by scott64bt** [comment]
I've edited that section and modified it to SQL Express 2012, which is the default download and a version we know users have in production. Please let me know if this is acceptable for you.

**Comment by scott64bt** [comment]
I recommitted those changes a few days ago- were my changes acceptable for you guys? Thanks - Scott

**Comment by gep13** [comment]
@scott64bt thanks for following up.  I am fairly sure that the suggestion was to completely remove mention of using SQL Express, in favour of a commercial version of SQL.  While people "might" use SQL Express in production, it certainly isn't recommended, and not something that we want to recommend our users do here.

/cc @ferventcoder for confirmation.

File: How-To-Host-Feed.md [hunk 2]
```diff
@@ -345,10 +345,10 @@ See https://inedo.com/support/documentation/proget/installation/installation-gui
...
-* Shared Storage: Appliance-based (such as NAS), or software-based (such as SMB). This will be shared across all Servers, so they will need to be able to read and write to it at high speed. Recommend appliance-based (NAS) with 3 times the total size of what you will store (and a backup) - 2-5TB+ space.
-* Database: SQL Server with Database Clustering Recommended. Assume high specs for SQL Server, pretty much the highest specs you can configure on a Windows machine. 16+ cores (CPU), 64GB+ RAM, 5TB+, etc
-* Load Balancer: Assume powerful - this will be software-based (such as nginx or HAProxy) or an appliance (such as F5 or Citrix).
-* May or may not support cross-datacenter replication.
+* Shared Storage: Shared Storage: Compatible with any type of common storage that web and indexing nodes can access, whether software-based (SAMBA share, Windows Server Storage Spaces, etc) or appliance-based (dedicated NAS). The only requirement is that the storage is readable and writeable by all nodes. Recommend appliance-based (NAS) with 3 times the total size of what you will store (and a backup) - 2–5TB+ space.
```
Comment (ferventcoder) [inline How-To-Host-Feed.md]: We don't spell out SMB anywhere in our documentation - let's keep that consistency.

File: How-To-Host-Feed.md [hunk 3]
```diff
@@ -345,10 +345,10 @@ See https://inedo.com/support/documentation/proget/installation/installation-gui
...
+* Database: SQL Server with Database Clustering Recommended. Assume higher than minimal specs for better High Availability performance.
```
Comment (ferventcoder) [inline How-To-Host-Feed.md]: Please leave our recommendations for SQL Server as they were with the actual values there.

File: How-To-Host-Feed.md [hunk 4]
```diff
@@ -332,11 +332,11 @@ See https://help.sonatype.com/repomanager3/high-availability-introduction for de
...
 * Windows Server
-* CPU - 4 cores (Recommend more)
-* RAM - 8GB+
-* Storage (HDD) - We recommend 2-5TB of free space for a local file store of artifacts (default install needs 20GB).
-* Database: SQL Server. Assume high specs for SQL Server, pretty much the highest specs you can configure on a Windows machine. 16+ cores (CPU), 64GB+ RAM, 5TB+, etc
-* Chocolatey Repository Type: Chocolatey or NuGet
+* CPU - 2 cores (4+ cores recommended for large instances)
+* RAM - 4GB+ (8GB recommended for large instances)
+* Storage (HDD) - We recommend 2-5TB of free space for a local file store of artifacts (default install needs 10GB).
+* Database: Higher specs may be required if creating more than a simple feed repository. Minimum, SQL Server Express 2012, (install available during download) 1GB RAM, 6GB disk space.
```
Comment (ferventcoder) [inline How-To-Host-Feed.md]: We are in the commercial section - organizations that are going to use these things are not concerned with Minimums and we are not talking minimums. We are talking Chocolatey's recommendations for folks to have a good time. Please leave this as it was.

**Comment by ferventcoder** [comment]
Ping - have not had any updates in awhile.

**Comment by ferventcoder** [comment]
Hi @scott64bt - not sure if you have had an opportunity to look over our notes and make adjustments. If you are not interested, simply mention as much and we will see what we can do to get what we can in. Thanks!

**Comment by ferventcoder** [comment]
Unfortunately, we have not seen progress to move this forward and thus must close the request at this time. If you wish to pick this up in the future, we'll be ready to review. Thanks.

**Pull request closed without merging**